spec.go

```diff
@@ -180,6 +180,55 @@ func parseFunctions(commands []xmlCommand) (specFunctions, error) {
 	return functions, nil
 }
 
+func parseOverloads(functions specFunctions, overloads xmlOverloads) (specFunctions, error) {
+	for _, overloadInfo := range overloads.List {
+		found := false
+		for key, function := range functions {
+			if key.name == overloadInfo.Name {
+				found = true
+				err := overloadFunction(function, overloadInfo)
+				if err != nil {
+					return nil, err
+				}
```

Comment:
break after finding & overloading?

Reply:
I wasn't sure whether a function could be in the list multiple times (in different API levels). I understand that you see this not the case - code refactored.